benchmarks/image-classification/rust-benchmark/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "image-classification-benchmark"
+version = "0.1.0"
+authors = ["The Bytecode Alliance Developers"]
+readme = "README.md"
+edition = "2018"
+publish = false
+
+[dependencies]
+image = { version = "0.23.14", default-features = false, features = ["gif", "jpeg", "ico", "png", "pnm", "tga", "tiff", "webp", "bmp", "hdr", "dxt", "dds", "farbfeld"]  }
```

Comment:
I think we're only using `"jpeg"` here, right?

Reply:
I suppose there's no reason currently to pull in the other image types.